cf-agent/retcode.c

```diff
@@ -41,17 +43,23 @@ bool VerifyCommandRetcode(EvalContext *ctx, int retcode, const Attributes *a, co
         char retcodeStr[PRINTSIZE(retcode)];
         xsnprintf(retcodeStr, sizeof(retcodeStr), "%d", retcode);
 
+        LogLevel info_or_verbose = LOG_LEVEL_INFO;
+        if (StringSafeEqual("commands", pp->parent_promise_type->name) && (!a->inform))
```

Comment:
Would be nice to add a comment here.

Reply:
@vpodzime https://github.com/cfengine/core/pull/3974